Its.Log/Its.Log.csproj

```diff
@@ -1,23 +1,17 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="12.0">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
```

Comment:
I would hope this doesn't need to target `net462`. Was there a specific issue you hit that required this?

Reply:
I am actually happy with just targeting net standard 2.0. I was thinking that some other people might want to use the 4.6.2 version because .Net Standard 2.0 is not fully compatible with .NET Framework versions lower than 4.7.2.

Reply:
I agree; net462 is desirable.